acceptance/replication_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Marc Berhault (marc@cockroachlabs.com)
+
+// +build acceptance
+
+package acceptance
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach/acceptance/localcluster"
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/storage/engine"
+	"github.com/cockroachdb/cockroach/util"
+	"github.com/cockroachdb/cockroach/util/log"
+)
+
+// makeDBClient creates a DB client for node 'i'.
+// It uses the cluster certs dir.
+func makeDBClient(cluster *localcluster.Cluster, node int) (*client.DB, error) {
+	// We always run these tests with certs.
+	db, err := client.Open("https://root@" +
```

Comment:
fair enough. I just like to wrap errors to get more information on where they come from.
eg: this would show me what it was caught at L44.